toolz/_signatures.py

```diff
@@ -664,7 +664,7 @@ def signature_or_spec(func):
         try:
             return inspect.getargspec(func)
         except TypeError as e:
```

Comment:
This patch makes `e` unused variable.
```suggestion
        except TypeError:
```